hardware-and-software-requirements.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Software and Hardware Recommendations
-summary: Learn the software and hardware recommendations for deploying and running TiDB.
+title: TiDB Software and Hardware Requirements
+summary: Learn the software and hardware requirements for deploying and running TiDB.
 aliases: ['/docs/dev/hardware-and-software-requirements/','/docs/dev/how-to/deploy/hardware-recommendations/']
 ---
 
-# Software and Hardware Recommendations
+# TiDB Software and Hardware Requirements
```

Comment:
Note: This is to keep wording consistent with that in https://docs.pingcap.com/tidb/dev/check-before-deployment and that in Chinese.